services/centralized-grafana/33.1.3/defaults/cm.yaml

```diff
@@ -2,15 +2,16 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: centralized-grafana-33.1.1-d2iq-defaults
+  name: centralized-grafana-33.1.3-d2iq-defaults
   namespace: ${releaseNamespace}
 data:
   values.yaml: |
     ---
+    fullnameOverride: "centralized-grafana"
     mesosphereResources:
       create: false
       ingressRBAC:
-        enabled: false
+        enabled: true
```

Comment:
FYI dkp-centralized-grafana-admin/view/edit will become

centralized-grafana-dkp-grafana-admin/view/edit

@gracedo

Reply:
I think that’s fine as long as we aren’t referencing the clusterroles anywhere (I don’t think so? But maybe worth searching docs)

Reply:
oops, forgot about the docs. They are actually mentioned in docs: https://docs.d2iq.com/dkp/kommander/2.1/operations/access-control/rbac/

I vote for using the new name but also open to change it back. I will open a docs PR if the new name is OK

Reply:
Thanks for finding that page! It actually looks like the kps role names might be the odd ones out as all others are dkp- prefixed. I wonder if we instead want to modify the kps chart clusterrole templates to use the prefix? So instead of kube-prometheus-stack-dkp-alertmanager-view, we could template it to dkp-kube-prometheus-stack-alertmanager-view instead? (dkp-<release name>-…)

Reply:
opened docs PR anyways for now: https://github.com/mesosphere/dcos-docs-site/pull/4180.
We can still rollback tomorrow to the old way if we determined that's the way to go.